README.rst

```diff
@@ -16,3 +16,5 @@
 with special focus on testing.  It is used in Zope, but it's not Zope
 specific at all.  For instance, it can be used to test or otherwise
 interact with any web site.
+
+Documentation is available at: http://zopetestbrowser.readthedocs.org
```

Comment:
Can you make this an https:// link please?

Reply:
Of course! Resolved in the amended commit.